controllers/configurationpolicy_controller.go

```diff
@@ -2982,6 +2982,11 @@ func handleKeys(
 				continue
 			}
 
+			// for ServiceAccounts, ignore "secrets" and "imagePullSecrets" fields, as these are managed by Kubernetes
+			if existingObj.GetKind() == "ServiceAccount" && (key == "secrets" || key == "imagePullSecrets") {
```

Comment:
(optional) There's a small chance there could be a `ServiceAccount` kind in a non-K8s core group, so we could add that check, but I don't think it's necessary.

Reply:
Would a check for `apiVersion == "v1"`  be sufficient for this?